cli/cdsctl/workflow.go

```diff
@@ -376,6 +382,9 @@ func workflowPullRun(c cli.Values) error {
 		if err := fi.Close(); err != nil {
 			return err
 		}
+		if !c.GetBool("quiet") {
+			fmt.Printf("File %s created\n", fname)
```

Comment:
why not just print the filename with the relative path from the working directory ? This way it can be easily used in a bash script

Reply:
yes